.gitignore

```diff
@@ -1,134 +1,17 @@
-.idea/
-build/
```

Comment:
Think we will want to keep these.

Reply:
These are still in the file, just in a different place

Reply:
Are you referring to `/dist` we still have our npm scripts that output to `build` so I think we should keep this . Unless im missing something else.